.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,46 @@
-<!--  **WIP-** prefix in title if still work in progress -->
+<!-- Common title prefixes/annotations:
+
+  WIP:
+  bugfix:
+  🏗️ maintenance:
+  UI:
```

Comment:
We already have the frontend tag...

Reply:
true. It might be handy to create the CHANGELOG text though